python/ray/_private/function_manager.py

```diff
@@ -382,7 +382,10 @@ def _load_function_from_local(self, function_descriptor):
 
         object = self.load_function_or_class_from_local(module_name, function_name)
         if object is not None:
-            function = object._function
+            if isinstance(object, RemoteFunction):
```

Comment:
Can you add a comment here to explain why we need this "if-else"?

Reply:
added